pom.xml

```diff
@@ -146,18 +145,6 @@
           </execution>
         </executions>
       </plugin>
-      <plugin>
```

Comment:
@hizmailovich what's wrong with it?

Reply:
@hizmailovich this plugin is more valuable than text blocks

WDYT?

Reply:
@hizmailovich it's time to make a decision

Reply:
Text blocks removed